packages/app/src/systems/Core/hooks/useTokensMethods.ts

```diff
@@ -29,7 +29,7 @@ export function useTokenMethods(tokenId: string) {
     async mint(gasLimit: bigint) {
       return contract.submitResult.mint(
         getOverrides({
-          variableOutputs: 1,
+          variableOutputs: 2,
```

Comment:
Here also should be 1, as mint function only outputs 1 type of asset

Reply:
✅